contributors/guide/contributor-cheatsheet/README-ja.md

```diff
@@ -290,6 +290,13 @@ git checkout -b myfeature
 [スカッシュコミット]の主な目的はきれいに読めるgitの履歴や加えられた変更のログを作成することです。通常これはPRの改定の最終段階に行われます。
 コミットをスカッシュするべきかわからない場合は、作業を止めてPRのレビューと承認を担当する他のコントリビューターの判断に任せることをおすすめします。
 
```

Comment:
In [original document](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet#squashing-commits), there is a sentence about `Perform an interactive rebase to choose which commits ...` on second paragraph.

Could you add translation for that?

Reply:
Nice catch, thanks.